.changeset/rich-insects-exercise.md

```diff
@@ -0,0 +1,12 @@
+---
+"@ledgerhq/coin-aptos": major
```

Comment:
@hedi-edelbloute I set Aptos coin module as `major`. Let me know if it makes sense.

Reply:
We didn't put standard yet, semver: major => breaking change (what does it mean, bridge interface stays the same ?) WDYT @sprohaszka-ledger

Reply:
Well, I am a little bit torn between the 2 as:
* There is no breaking change in the interface --> not a major version
* There is the token support in the coin module --> a major feature

I would go with the `major` here as there might be side effects and therefor "invisible" breaking change.